pages/linux/tmt-run.md

```diff
@@ -0,0 +1,37 @@
+# tmt run
+
+> Execute tmt test steps. By default, all steps are run.
+> See also: `tmt`, `tmt-try`.
```

Comment:
```suggestion
```
We do not use this syntax to reference other subcommands.

Reply:
Thanks for the review!  
So, what syntax would you like us to use?
[contributing doc](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md) mentions `For page descriptions, you can additionally use See also: `command`.`  
The other contributing doc page states:
```
The following methods can be used to reference subcommands:

 You can use See also: `command1`, `command2`. template to reference similar commands, aliases and subcommands.
 ```

Reply:
oh I see the suggestion below: `Some subcommands such as `run`, `try`, etc. have their own usage documentation.`